index.html

```diff
@@ -1,204 +1,310 @@
-</li>
-  </ul>
-
-  <ul class="html-css">
-    <li class="html-container">
-   <img  class ="html" src="portfolioimgs/HTML.png" alt="html"/> <span class="html-li">HTML</span>
-    </li>
-  <li class="css-container">
-   <img class="css" src="portfolioimgs/CSS.png" alt="css"/> <span class="css-li">CSS</span>
-  </li>
-  </ul>
-  <ul class="Frameworks">
-    <li>Frameworks</li>
-   <li> <img class="scrollDown2" src="portfolioimgs/down2.png" alt="down">
-  </li>
-  </ul>
-
-   <ul class="Skills">
-    <li>Skills</li>
-    <li><img class="scrollDown2" src="portfolioimgs/down2.png" alt="down"></li>
-  
-   </ul>
-
-</div>
+        <div class="resume">
+          <button type="button">Get my resume</button>
+          <ul class="languages">
+            <li>Languages</li>
+            <li>
+              <img class="scrollDown" src="portfolioimgs/down.png" alt="down" />
+            </li>
+          </ul>
 
-</section>
+          <ul class="html-css">
+            <li class="html-container">
+              <img class="html" src="portfolioimgs/HTML.png" alt="html" />
+              <span class="html-li">HTML</span>
+            </li>
+            <li class="css-container">
+              <img class="css" src="portfolioimgs/CSS.png" alt="css" />
+              <span class="css-li">CSS</span>
+            </li>
+          </ul>
+          <ul class="Frameworks">
+            <li>Frameworks</li>
+            <li>
+              <img
+                class="scrollDown2"
+                src="portfolioimgs/down2.png"
+                alt="down"
+              />
+            </li>
+          </ul>
 
-    
+          <ul class="Skills">
+            <li>Skills</li>
+            <li>
+              <img
+                class="scrollDown2"
+                src="portfolioimgs/down2.png"
+                alt="down"
+              />
+            </li>
+          </ul>
+        </div>
+      </section>
 
-</div>
-</body>
-</html>
\ No newline at end of file
+      <section class="contact-me">
+          <h5 class="section-header">Contact me</h5>
+          <p class="text-contact">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form>
+            <input
+              type="text"
+              name="name"
+              value="Yeremias"
+              required
+              size="30"
+              placeholder="Name"
+              maxlength="10"
+            />
+            <input
+              type="email"
+              name="email"
+              value="natajayanj14@gmail.com"
+              placeholder="Email"
+              required
+              size="30"
+            />
+            <textarea
+              name="message"
+              id="message"
+              cols="30"
+              rows="10"
+              placeholder="Write you r message here"
+            
```

Comment:
- [ ] Please make sure to avoid blank spaces, empty lines, or commented code, so this will help you to keep your code clean and organized, and it's part of the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).